lib/util/github/graphql/cache-strategies/abstract-cache-strategy.ts

```diff
@@ -23,7 +23,7 @@ export abstract class AbstractGithubGraphqlCacheStrategy<
   /**
    * The time which is used during single cache access cycle.
    */
-  protected readonly now = DateTime.now();
+  protected readonly now = DateTime.now().toUTC();
```

Comment:
`DateTime.utc()`?

Reply:
Looking at [the Luxon doc](https://moment.github.io/luxon/api-docs/index.html), I see `toUTC()`? I do not see an instance method called `utc()`. I do see a static function called `utc()`.